plugins/generator-1.15.2/forge-1.15.2/biome.definition.yaml

```diff
@@ -1,3 +1,7 @@
 templates:
   - template: biome.java.ftl
-    name: "@SRCROOT/@BASEPACKAGEPATH/world/biome/@NAMEBiome.java"
\ No newline at end of file
+    name: "@SRCROOT/@BASEPACKAGEPATH/world/biome/@NAMEBiome.java"
+
+localizationkeys:

```

Comment:
Does biome name do anything in 1.14.4 and 1.15.2? If not, this might be moved into 1.16.x branch instead of in case 2020.5 will not be 1.16.x yet.

Reply:
As I explained in the description, they are used at some places like when you choose a biome for your Buffet (or Single Biome) world.

Reply:
Sorry, my bad.